sfcasts/factory-families.md

```diff
@@ -78,7 +78,7 @@ and add a `break` at the end. *Awesome*.
 
 Now, you may be thinking "What if the `UltimateAttackTypeFactory` has
 dependencies?" or "What if it's not that simple to instantiate?", and that's a
-valid concern. A way to solve that is by doing the same thing we did with the
+valid concern. A way to solve this is the same as what we discussed with
```

Comment:
We only discussed _how_ we could solve this in that chapter. We didn't actually _do_ it.